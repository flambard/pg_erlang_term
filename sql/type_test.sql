CREATE TABLE slask (id serial, term erlang_term);


INSERT INTO slask (term)
VALUES ('[]'), ('erlang'), ('{tusan, ["vad", "fin" | "t"]}');


INSERT INTO slask (term)
VALUES ('[inte erlang}');


SELECT *
FROM slask;