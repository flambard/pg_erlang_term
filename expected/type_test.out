CREATE TABLE slask (id serial, term erlang_term);
INSERT INTO slask (term)
VALUES ('[]'), ('erlang'), ('{tusan, ["vad", "fin" | "t"]}');
INSERT INTO slask (term)
VALUES ('[inte erlang}');
ERROR:  invalid erlang term syntax
LINE 2: VALUES ('[inte erlang}');
                ^
SELECT *
FROM slask;
 id |             term             
----+------------------------------
  1 | []
  2 | erlang
  3 | {tusan, ["vad", "fin", "t"]}
(3 rows)

